plugins/headlamp-plugin/package.json

```diff
@@ -3,6 +3,9 @@
   "version": "0.3.2",
   "description": "The needed infrastructure for building Headlamp plugins.",
   "bin": "bin/headlamp-plugin.js",
+  "scripts": {
```

Comment:
Seems like this needs to be run any time there is a change in the frontend (types).

So we should add a CI check which runs `npm run build` and checks for git changes.

Reply:
@illume , I have added the CI check for `npm run build`. You mentioned you had done something similar to what's needed for the type check, so maybe you can add that commit or point me in the right direction.

Reply:
I actually removed the patch now because I forgot that we're not tracking the generated types. We should still add the check for package versions.